dapps/marketplace/src/utils/pasteIntoInput.js

```diff
@@ -0,0 +1,17 @@
+export default function pasteIntoInput(el, text) {
+  el.focus()
+  if (
+    typeof el.selectionStart == 'number' &&
+    typeof el.selectionEnd == 'number'
+  ) {
+    const val = el.value
+    const selStart = el.selectionStart
+    el.value = val.slice(0, selStart) + text + val.slice(el.selectionEnd)
+    el.selectionEnd = el.selectionStart = selStart + text.length
+  } else if (typeof document.selection != 'undefined') {
```

Comment:
nit: add a short comment either here in the description of the pasteIntoInput method why in some cases el.selectionStart/End would not be defined (I assume it is not supported by all browsers ?).

Reply:
done